src/rhelocator/api/schema/health.yml

```diff
@@ -0,0 +1,29 @@
+---
+definitions:
```

Comment:
Can I replace this with the following?
```yaml
 definitions:
  type: object
  properties:
    message:
      type: string
```

Reply:
Yeah that would work. You could also omit the definitions block if you preferre:
```
responses:
  200:
    description:
    schema:
      type: object
        properties:
          message:
            type: string
    examples:
          {
            "message": "ok"
          }
```